postgresql/adapter_test.go

```diff
@@ -207,7 +407,7 @@ func TestOptionTypes(t *testing.T) {
 	type optionType struct {
 		ID       int64                  `db:"id,omitempty"`
 		Name     string                 `db:"name"`
-		Tags     []string               `db:"tags,stringarray"`
+		Tags     StringArray            `db:"tags"`
```

Comment:
how old is your lib/pq? https://github.com/lib/pq/commit/edf9f9f99e9cbeec669f15fed9a72f23ed193b8c